infra-templates/alidns/2/rancher-compose.yml

```diff
@@ -1,7 +1,7 @@
 # notemplating
 .catalog:
   name: "Alibaba Cloud DNS"
-  version: "v0.2.0"
+  version: "v0.3.0"
   description: "Rancher External DNS service powered by Alibaba Cloud"
   minimum_rancher_version: v1.6.8-rc1
```

Comment:
can we update this to what will be the next release as we will only be testing against that version? 

`v1.6.12-rc1`